packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -400,6 +400,14 @@ class _CupertinoAlertDialogState extends State<CupertinoAlertDialog> {
 /// Additionally, the white paint can be disabled to render a blurred rounded
 /// rectangle without any color (similar to iOS's volume control popup).
 ///
+/// {@tool dartpad}
+/// This sample shows how to use a [CupertinoPopupSurface].
+/// The [CupertinoPopupSurface] shows a model popup from the bottom of the screen.
```

Comment:
```suggestion
/// This sample shows how to use a [CupertinoPopupSurface]. The [CupertinoPopupSurface]
/// shows a model popup from the bottom of the screen.
```
This line break looks unusual here with short sentence.